### PR TITLE
[POC] moving admin

### PR DESCRIPTION
**Pull request by ElectricMaxxx**
Moved:
- [x] seo-bundle
- [x] menu-bundle
- [x] core-bundle
- [x] content-bundle
- [x] block-bundle
- [x] routing-bundle

### CODE REVIEW

**Comment by ElectricMaxxx** [comment]
It seems as the extraction of seo admin stuff is working now. I had one issue in the phpcr-odm-admin-bundle => https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/pull/385 But I think The Seo PR is ready to review.

File: composer.json [hunk 1]
```diff
@@ -16,6 +16,12 @@
     "autoload-dev": {
         "classmap": [ "app/tests/WebTestCase.php" ]
     },
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "git@github.com:ElectricMaxxx/sonata-admin-integration-bundle.git"
+        }
+    ],
```
Comment (WouterJ) [inline composer.json]: this is not needed. The branch lives in the main repository, so no need to configure composer to use your fork.

Comment (ElectricMaxxx) [inline composer.json]: I need to move it today until that it lives in my space :-)
Will move when we have the name.

**Comment by ElectricMaxxx** [comment]
I think we have them all. We moved all admins into `sonata-admin-integration-bundle`. Before saying "yea we did" we should use them all in the sandbox, right?

**Comment by dbu** [comment]
agreed yes, we should update the sandbox to use the new admins. that
will also reveal if there are any glitches or errors left

**Comment by ElectricMaxxx** [comment]
So ...
I started the branch again. But i have got problems to install sonata-admin in version ^3.6.0

**Comment by ElectricMaxxx** [comment]
@wouterj can you give me a hand with the `sonata-project/admin-bundle: ^3.6.0` task you gave me through sonata-admin-integration-bundle. Can't get it updated :-(

File: composer.json [hunk 2]
```diff
@@ -39,15 +39,21 @@
 
         "sonata-project/cache-bundle": "^2.1",
         "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
-        "sonata-project/admin-bundle": "^3.2",
+        "sonata-project/admin-bundle": "3.2 as ^3.6.0",
```
Comment (wouterj) [inline composer.json]: this is incorrect. Why not doing `^3.6` here?

File: app/config/config.yml [hunk 3]
```diff
@@ -127,6 +127,15 @@ cmf_resource_rest:
...
         - { repository: default, enhancer: sonata_admin }
 
+cmf_sonata_admin_integration:
+  bundles:
+    seo: ~
+    menu: ~
+    core: ~
+    content: ~
+    block: ~
+    routing: ~
```
Comment (wouterj) [inline app/config/config.yml]: use 4 spaces indentation

File: composer.json (hunk 2 above)
Comment (wouterj) [inline composer.json]: @ElectricMaxxx why this as? Why not doing `"sonata-project/admin-bundle": "^3.6"`? This still gives us SonataAdminBundle 3.2, which doesn't work witht the SonataIntegrationBundle.

**Comment by ElectricMaxxx** [comment]
Now is the first time we have to use a preprocessor for the content-bundle's `document_class` parameter, cause we defined it in configuration before to add the seo stuff working on StaticContent

btw @wouterj because of that:

```
maximilian@maximilian-ThinkPad-T540p:~/OpenSource/Cmf/cmf-sandbox$ composer require sonata-project/admin-bundle:"^3.6.0"
You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install sonata-project/admin-bundle 3.10.2
    - Conclusion: don't install sonata-project/admin-bundle 3.10.1
    - Conclusion: don't install sonata-project/admin-bundle 3.10.0
    - Conclusion: don't install sonata-project/admin-bundle 3.9.0
    - Conclusion: don't install sonata-project/admin-bundle 3.8.0
    - Conclusion: don't install sonata-project/admin-bundle 3.7.1
    - Conclusion: don't install sonata-project/admin-bundle 3.7.0
    - Conclusion: don't install sonata-project/admin-bundle 3.6.0
    - Conclusion: remove sonata-project/block-bundle 3.1.1
    - Installation request for sonata-project/core-bundle (locked at 3.0.3) -> satisfiable by sonata-project/core-bundle[3.0.3].
    - Installation request for sonata-project/admin-bundle ^3.6.0 -> satisfiable by sonata-project/admin-bundle[3.10.0, 3.10.1, 3.10.2, 3.6.0, 3.7.0, 3.7.1, 3.8.0, 3.9.0, 3.x-dev].
    - Conclusion: don't install sonata-project/block-bundle 3.1.1
    - sonata-project/admin-bundle 3.x-dev requires sonata-project/block-bundle ^3.2 -> satisfiable by sonata-project/block-bundle[3.2.0, 3.x-dev].
    - Can only install one of: sonata-project/block-bundle[3.2.0, 3.1.1].
    - Can only install one of: sonata-project/block-bundle[3.x-dev, 3.1.1].
    - Installation request for sonata-project/block-bundle (locked at 3.1.1) -> satisfiable by sonata-project/block-bundle[3.1.1].


Installation failed, reverting ./composer.json to its original content.
```

File: composer.json [hunk 4]
```diff
@@ -39,15 +39,21 @@
...
         "lunetics/locale-bundle": "^2.5",
         "burgov/key-value-form-bundle": "^1.4",
-        "egeloen/ckeditor-bundle": "^4.0"
+        "egeloen/ckeditor-bundle": "^4.0",
+        "symfony-cmf/sonata-admin-integration-bundle": "^1.0",
+        "symfony-cmf/routing-bundle": "^2.0",
+        "symfony-cmf/menu-bundle": "^2.0",
+        "symfony-cmf/block-bundle": "^2.0",
+        "symfony-cmf/core-bundle": "^2.0",
+        "symfony-cmf/content-bundle": "^2.0"
```
Comment (wouterj) [inline composer.json]: All these bundles are required in `symfony-cmf/symfony-cmf`, so they can be removed.

**Comment by wouterj** [comment]
@ElectricMaxxx maybe a caching issue on your laptop (try removing `composer.lock` and `vendor`). I've fetched this branch, updated the composer.json file to use `^3.6` and ran `composer update`. This perfectly gives me 3.10.2 (the latest version) without any error.

**Comment by ElectricMaxxx** [comment]
@wouterj  didn't made an `update` tried to get explicit versions by install and require into the `composer.lock`

**Comment by ElectricMaxxx** [comment]
I am getting to a point where i am blocked by https://github.com/symfony-cmf/sonata-admin-integration-bundle/pull/35 (with https://github.com/symfony-cmf/sonata-admin-integration-bundle/pull/39)

**Comment by ElectricMaxxx** [comment]
Except the little bug https://github.com/symfony-cmf/sonata-admin-integration-bundle/issues/44 and the work in https://github.com/symfony-cmf/sonata-admin-integration-bundle/pull/43 it looks really good. Admin works as far as i can see. :-)

**Comment by wouterj** [comment]
Ready to merge

**Comment by dbu** [comment]
awesome!

**Pull request merged**